src/helpers/similarity.ts

```diff
@@ -0,0 +1,106 @@
+import { getLogger } from "../bindings";
+import axios, { AxiosError } from "axios";
+import { ajv } from "../utils";
+import { Static, Type } from "@sinclair/typebox";
+import { backOff } from "exponential-backoff";
+
+export const extractImportantWords = async (content: string): Promise<string[]> => {
+  const res = await getAnswerFromChatGPT(
+    '',
+    `${
+      process.env.CHATGPT_USER_PROMPT_FOR_IMPORTANT_WORDS ||
+      "I need your help to find important words (e.g. unique adjectives) from github issue below and I want to parse them easily so please separate them using #(No other contexts needed). Please separate the words by # so I can parse them easily. Please answer simply as I only need the important words. Here is the issue content.\n"
+    } '${content}'`,
+    parseFloat(process.env.IMPORTANT_WORDS_AI_TEMPERATURE || "0")
+  );
+  console.log(res);
+  return res.split(/[,# ]/);
+};
+
+export const measureSimilarity = async (first: string, second: string): Promise<number> => {
```

Comment:
```suggestion
export const measureSimilarity = async (first: Issue, second: Issue): Promise<number> => {
```

I think it's better to let the function extract whatever it needs from the issue